DataConnectors/Senserva/SenservaPro.json

```diff
@@ -0,0 +1,159 @@
+{
+	"id": "SenservaPro",
+	"title": "SenservaPro",
+	"publisher": "Senserva",
+	"descriptionMarkdown": "The SenservaPro data connector provides a viewing experience for your SenservaPro scanning logs. View dashboards of your data, use queries to hunt & explore, and create custom alerts for your needs. Start your security journey today",
+	"additionalRequirementBanner": "No additional requirements needed",
```

Comment:
If there is no additional requirement. please remove this tag.

Thanks,
Aditi Maheshwari

Reply:
@v-admahe This change has been made